k8s/charts/msf-xk8s/templates/prometheus/alertingRules.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: msf-xk8s-service-monitoring-rules
+  namespace: monitoring
+  labels:
+    app: msf-xk8s
+    component: lcm
+    team: msf
+    chart: msf-xk8s
+    release: {{ .Release.Name }}
+data:
+  msf-xk8s-service-monitoring-rules.yaml: |+
+    groups:
+    - name: msf-xk8s-service-monitoring-rules
+      rules:
+      - record: "container_pod:xk8s_pod_container_status_restarts:increase10m"
+        expr: increase(kube_pod_container_status_restarts_total{namespace='lcm'}[10m])
+      - alert: "[MSF XK8S] Pod has too many restarts on cluster={{ .Values.clusterId }}"
+        expr: container_pod:xk8s_pod_container_status_restarts:increase10m >= 1
+        labels:
+          severity: warning
+          team: hds # Used to reflect Icinga style alerting
+          cluster_id: {{ .Values.clusterId }}
+        annotations:
+          description: "There is more than 0 restarts of {{`{{ $labels.pod }}`}} pod in the last 10 minutes"
+          summary: "{{`{{ $labels.pod }}`}} pod has too many restarts"
+      - alert: "[MSF XK8S] Pod has too many restarts on cluster={{ .Values.clusterId }}"
+        expr: container_pod:xk8s_pod_container_status_restarts:increase10m >= 2
+        labels:
+          severity: critical
+          team: hds # Used to reflect Icinga style alerting
+          cluster_id: {{ .Values.clusterId }}
+        annotations:
+          description: "There is more than 1 restart of {{`{{ $labels.pod }}`}} pod in the last 10 minutes"
+          summary: "{{`{{ $labels.pod }}`}} pod has too many restarts"
+      - record: "container_pod:xk8s_pod_container_status_oomkilled:increase10m"
+        expr: increase(kube_pod_container_status_terminated_reason{namespace='{{ .Release.Namespace }}', reason='OOMKilled', pod=~'^xk8s.*'}[10m])
```

Comment:
It makes no sense to add `pod=~'^xk8s.*'` as your pods do not have any human friendly names containing such string:
```
[root@na1-k8s-master03:/home/milan.sladky] kubectl get pods -n lcm 
NAME                                                             READY   STATUS      RESTARTS   AGE
0016a531f72f7b6b7ef7d7283095de55fa20603ad212859c00000820-66kbv   0/1     Completed   0          11d
16d9b22dbf7fbdd0c6fe7e46c15d0ed78678043cec8689ff00000820-trrwl   0/1     Completed   0          14d
1d49a420efc6ecb9bd8ad9cf7127bc97c526ccae59ff447b00000820-cwq6v   0/1     Error       0          14d
2eea2e79f2f6eb8e22ea258258a557e1e8a0c7c0de68eb7000000820-2pwdx   0/1     Completed   0          11d
414e68e0e60c0e059d34da1f8c4adb24dec3423df528260700000820-bqwfs   0/1     Completed   0          44h
b2c6e28799c8967569b0410b26f39e2ae0a1b32fba8a4bc700000820-zmlgf   0/1     Error       0          11d
b577ea144a1fe5e48c36bd1f5be401332ef1534d39c4f7f700000820-c2r56   0/1     Error       0          44h
c6c5424c3a160c764f911570435cec89fcb23475fbaf18a500000820-pwc7r   0/1     Error       0          11d
```